hoot-core/src/main/cpp/hoot/core/cmd/ConflateCmd.cpp

```diff
@@ -481,37 +481,9 @@ void ConflateCmd::_updatePostConfigOptionsForAttributeConflation()
   // we have a better solution for changing these opts in place.
   if (ConfigOptions().getHighwayMergeTagsOnly())
   {
-    // If we're running Attribute Conflation and removing building relations, we need to remove them
```

Comment:
These were all hack-ish config opt workarounds to make AC do building conflation the way Tradecraft wanted. This PR is the result of fixing the underlying problem with building merging and makes these option unnecessary